lib/rubocop/rspec/support.rb

```diff
@@ -11,4 +11,18 @@
 RSpec.configure do |config|
   config.include CopHelper
   config.include HostEnvironmentSimulatorHelper
+  config.include_context 'config', :config
+  config.include_context 'isolated environment', :isolated_environment
+  config.include_context 'maintain registry', :restore_registry
+  config.include_context 'ruby 2.0', :ruby20
+  config.include_context 'ruby 2.1', :ruby21
+  config.include_context 'ruby 2.2', :ruby22
+  config.include_context 'ruby 2.3', :ruby23
+  config.include_context 'ruby 2.4', :ruby24
+  config.include_context 'ruby 2.5', :ruby25
```

Comment:
Oh, well... :D I should take a closer look there - I just noticed that @koic had some reservations about it. 

Probably in RuboCop 2.0 we'll drop all support for anything before Ruby 3.0 (or at least older than 2.7), but there's still plenty of time until we get there.

Reply:
I think RuboCop 2.0 should also support from Ruby 2.0 to 2.7.

I've seen some use cases where "RuboCop dropped the analysis version, which changed the Ruby minimum requirement version of Gem". In fact, I've heard cases of upgrading RuboCop because `TargetRubyVersion` can analyze down to Ruby 2.0.

So, old analysis version support (existing feature) is not hard maintenance and the maintenance will lead to the happiness of gem developers I think.